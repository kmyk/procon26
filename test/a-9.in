11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111110011111111011111111111111
11111111011111111011111111111111
11111100000000000000011111111111
11111111011111111011111111111111
11111111011111111001111111111111
11111111111111111011111111111111
11111111111111111011111111111111
11111111111111111001111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111
11111111111111111111111111111111

3
00000000
00000000
00101000
11111000
00100000
00100000
00100000
00100000

00000000
00000000
00100000
01110000
00100000
00100000
00100000
00100000

00000000
00000000
00100001
01111111
00100000
00100000
00100000
00100000
