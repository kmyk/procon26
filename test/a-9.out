5 2 H 270

13 5 H 90
